tsconfig.json

```diff
@@ -1,17 +1,12 @@
 {
   "compilerOptions": {
-    "lib": ["es6"],
+    "lib": ["dom", "es2017"],
     "module": "commonjs",
     "noImplicitAny": false,
     "outDir": ".tmp",
+    "resolveJsonModule": true,
```

Comment:
What is this necessary for?

Reply:
There's an import of a JSON file in one of the test files (`spec/providers/https.spec.ts`). This configuration is not used for production build.

Reply:
Ah is it this one: `import * as mocks from '../fixtures/credential/key.json';`? I'm all for adding better type support for imported jsons, so I don't have problems adding this in.